common/defs-flags.go

```diff
@@ -1,10 +1,12 @@
 package common
+
 const (
 	// Shared flag definitions
 	FlagEnvDefinition = "Environment to read environmental variables from."
 	FlagAddrDefinition = "The host and port (formatted as host:port) for the Temporal Frontend Service."
 	FlagNSAliasDefinition = "Identifies a Namespace in the Temporal Workflow."
 	FlagMetadataDefinition = "Contains gRPC metadata to send with requests (formatted as key=value). Values must be in a valid JSON format."
+	FlagTLSDefinition = "Enable TLS"
```

Comment:
we should elaborate here for users, maybe
```suggestion
	FlagTLSDefinition = "Enable TLS encryption without additional options such as mTLS or client certificates"
```